index.html

```diff
@@ -172,6 +173,12 @@ <h2>Definitions</h2>
         When set, this [=preference=] indicates that the person expects to browse the Web with
         [=do-not-sell-or-share interactions=].
       </p>
+      <p>
+        <dfn data-lt="cross-organization ad targeting|">Cross-organization targeted
+        advertising</dfn> means showing a person advertisements, where the advertisement is selected
+        based on data about that person that was gathered from organizations beyond just the one
+        they're interacting with when they see the advertisement.
+      </p>
```

Comment:
```suggestion
      <p>
        <dfn data-lt="cross-party ad targeting|">Cross-party targeted
        advertising</dfn> means showing a person advertisements, where the advertisement is selected
        based on data about that person that was gathered from any party other than the
        one the person intends to interact with when they are shown the advertisement.
      </p>
```
s/organization/party/, and tightly mirror definition to `do-not-sell-or-share interaction`